cfgov/v1/wagtail_hooks.py

```diff
@@ -46,12 +46,14 @@ def share_the_page(request, page):
 @hooks.register('after_delete_page')
 def log_page_deletion(request, page):
     logger.warning(
-        u'User {user} with ID {user_id} deleted page {title} with ID {page_id}'.format(
+        u'User {user} with ID {user_id} deleted page {title} with ID {page_id} at URL {url}'.format(
             user=request.user,
             user_id=request.user.id,
             title=page.title,
-            page_id=page.id)
+            page_id=page.id,
+            url=page.url,
```

Comment:
should we capture `page.url_path` instead, to cover the deletion of archived pages, which don't have urls?